### PR TITLE
Implement host show.

### PR DESCRIPTION
**Pull request by dmtucker**
`yaas host list | xargs yaas host show --hardware` works well.

### CODE REVIEW

File: yaas/host.py [hunk 1]
```diff
@@ -1,4 +1,87 @@
...
+        '--hardware',
+        action='store_true',
+        default=False,
+        help="Print hardware specs.")
+    subargs, extra = parser.parse_known_args(args)
+    for host_name in extra:
+        response = requests.get(
+            config.href('/api/v1/hosts/{host}'.format(host=host_name)),
+            **config.requests_opts())
+        if config.args.raw:
```
Comment (karth295) [inline yaas/host.py:89]: You should deal with failure cases (e.g. 404 host not found) before this. Can we do something like:

if not response.success:
    raise YaasException(response.json()["message"])

In **main**.py, we could catch the exception and print the message.

File: yaas/host.py [hunk 2]
```diff
@@ -1,4 +1,87 @@
...
+        subparsers.choices[subargs.subcommand],
+        extra)
+
+
+def _list(parser, args):
+    """ List all registered agents. """
+    response = requests.get(
+        config.href('/api/v1/hosts'),
+        **config.requests_opts())
+    if config.args.raw:
```
Comment (karth295) [inline yaas/host.py:57]: You should deal with failure cases before this, even though there shouldn't be any. See comment below

Comment (dmtucker) [inline yaas/host.py:57]: Fixed in #9

File: yaas/host.py (hunk 1 above)
Comment (dmtucker) [inline yaas/host.py:89]: This commit alone allows `HTTPError`s to bubble up to the user, but #13 will fix that.

**Pull request merged**